tests/src/test/scala/cats/tests/StateTTests.scala

```diff
@@ -114,14 +114,62 @@ class StateTTests extends CatsSuite {
     got should === (expected)
   }
 
+
+  implicit val iso = CartesianTests.Isomorphisms.invariant[StateT[ListWrapper, Int, ?]](StateT.catsDataFunctorForStateT(ListWrapper.monad))
+
+  {
+    // F has a Monad
+    implicit val F = ListWrapper.monad
+
+    checkAll("StateT[ListWrapper, Int, Int]", MonadStateTests[StateT[ListWrapper, Int, ?], Int].monadState[Int, Int, Int])
+    // checkAll("MonadState[StateT[ListWrapper, Int, ?], Int]", SerializableTests.serializable(MonadState[StateT[ListWrapper, Int, ?], Int]))
+    checkAll("MonadState[StateT[List, Int, ?], Int]", SerializableTests.serializable(MonadState[StateT[List, Int, ?], Int]))
```

Comment:
The serializable tests fail for `ListWrapper` and `List`, `Option` which was tested before, still works.